src/public/media.ts

```diff
@@ -437,6 +438,12 @@ export namespace media {
     }
     ensureInitialized(FrameContexts.content, FrameContexts.task);
 
+    if (!GlobalVars.isFramelessWindow) {
+      const notSupportedError: SdkError = { errorCode: ErrorCode.NOT_SUPPORTED_ON_PLATFORM };
+      callback(notSupportedError, undefined);
```

Comment:
Please make it null instead of undefined

Reply:
done.